data/language/kr.js

```diff
@@ -105,45 +105,45 @@ var language = [
 	{ id: "orderbypartymember", string: "파티 멤버 순으로 정렬" },
 	{ id: "order", string: "우선순위" },
 	{ id: "partyorder", string: "파티 우선순위 (드래그/드롭)" },
-	{ id: "mode", string: "Mode" },
-	{ id: "add", string: "Add" },
-	{ id: "edit", string: "Edit" },
-	{ id: "delete", string: "Delete" },
-	{ id: "save", string: "Save" },
-	{ id: "noabilitiesfound", string: "No abilities found..." },
-	{ id: "search", string: "Search" },
+	{ id: "mode", string: "모드" },
+	{ id: "add", string: "추가" },
+	{ id: "edit", string: "수정" },
+	{ id: "delete", string: "삭제" },
+	{ id: "save", string: "저장" },
+	{ id: "noabilitiesfound", string: "해당 기술이 없습니다..." },
+	{ id: "search", string: "검색" },
 	{ id: "id", string: "ID" },
-	{ id: "jobrole", string: "Job / Role" },
+	{ id: "jobrole", string: "직업 / 역할" },
 	{ id: "pvp", string: "PvP" },
-	{ id: "usewebtts", string: "Use Baidu TTS" },
+	{ id: "usewebtts", string: "바이두 TTS 사용" },
 	{ id: "jobthresholds", string: "Job Thresholds" },
```

Comment:
Can you explain about `Job Thresholds`? I didn't touch this because I didn't know what it was.